### PR TITLE
Create third-party-tools.zh-cn.md

### PR DESCRIPTION
**Pull request by maikebing**
Translated the third-party-tools.zh-cn

### CODE REVIEW

**Review by lunny — APPROVED** [review]

**Comment by lunny** [comment]
but the time is not right.

File: docs/content/doc/advanced/third-party-tools.zh-cn.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+# 第三方工具列表
+**注意:** 这些工具并没有经过Gitea的检验，在这里列出它们只是为了便捷.
+
+*此列表并不是完整的列表，可以随时咨询如何添加!*
+
+### 持续集成
+[BuildKite 连接器](https://github.com/techknowlogick/gitea-buildkite-connector)  
+[Jenkins 插件](https://github.com/jenkinsci/gitea-plugin)  
+[[Gitea搭配Drone](https://docs.drone.io/installation/gitea/)
```
Comment (jolheiser) [inline docs/content/doc/advanced/third-party-tools.zh-cn.md]: Duplicate opening bracket.

**Review by jolheiser — CHANGES_REQUESTED** [review]
Small nit

File: docs/content/doc/advanced/third-party-tools.zh-cn.md (hunk 1 above)
Comment (maikebing) [reply docs/content/doc/advanced/third-party-tools.zh-cn.md]: OK！

File: docs/content/doc/advanced/third-party-tools.zh-cn.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+---
+date: "2019-03-11T21:45:00+00:00"
+title: "高级: 第三方工具"
```
Comment (BetaCat0) [inline docs/content/doc/advanced/third-party-tools.zh-cn.md:3]: “高级”换做“进阶”看起来似乎顺畅一些😀

**Comment by 6543** [comment]
i trust @lunny so ...:

**Review by 6543 — APPROVED** [review]

**Pull request merged**